pkg/kubernetes/status.go

```diff
@@ -96,7 +96,7 @@ func (s *StatusClient) Status() ([]StatusOutput, error) {
 			namespace := pod.GetNamespace()
 			age := age.GetAge(pod.CreationTimestamp.Time)
 			created := pod.CreationTimestamp.Format("2006-01-02 15:04.05")
-			version := image[strings.IndexAny(image, ":")+1:]
+			version := image[strings.LastIndexAny(image, ":")+1:]
```

Comment:
https://pkg.go.dev/strings#LastIndexAny

There seems to be a change in behavior. If image is `1.8.0` (i.e. without a ":"), `IndexAny` will return `-1`, and hence version will be `1.8.0`. With the change, `LastIndexAny` will return 4, and it will throw an error on `image[5]` (index out of bounds).

We should take care of this and add a unit test.

Reply:
Will add unit test for this.. but here `LastIndexAny` will also return `-1` . see this - 

```
    s := "1.8.0"
    s1 := "xyz:dapr:1.8.0"
    fmt.Println(s[strings.LastIndexAny(s, ":") + 1:]) // 1.8.0
    fmt.Println(s[strings.IndexAny(s, ":") + 1:])    // 1.8.0
    
    fmt.Println(s1[strings.LastIndexAny(s1, ":") + 1:]) // 1.8.0
    fmt.Println(s1[strings.IndexAny(s1, ":") + 1:])    // dapr:1.8.0
```

Reply:
Ah, "the index of the last instance of **any Unicode code point** from chars in s", missed the `any`. Should we/can we use `LastIndex` index instead? It does a simple substring lookup.

Reply:
Yeah, `LastIndex` seems simpler. Let me update this.